### PR TITLE
DIGI_6_ CreateMenuitem

### PR DESCRIPTION
**Pull request by praveen957**
1.added MenuItemDataModel
2.added MenuItemController

### CODE REVIEW

File: DigiOrderApi/DigiOrderApi/Controllers/MenuItemController.cs [hunk 1]
```diff
@@ -1,13 +1,40 @@
...
 {
     public class MenuItemController : ApiController
     {
+        MenuItemDataModel menuItemDataModel = null;
+        DataTable dtRecords = null;
+
+        //Author: praveen kumar
+        //07-01-2020
+        //Description: Post a MenuItem
+        //Reviewer
```
Comment (anebhargavkrishna) [inline DigiOrderApi/DigiOrderApi/Controllers/MenuItemController.cs:20]: Add Reviewer Name

File: DigiOrderApi/DigiOrderApi/Controllers/MenuItemController.cs [hunk 2]
```diff
@@ -1,13 +1,40 @@
...
+        {
+            try
+            {
+                menuItemDataModel = new MenuItemDataModel();
+                dtRecords = menuItemDataModel.createMenuItem(data);
+                dtRecords.TableName = "CATEGORY";
+            }
+            catch (Exception EX)
+            {
+                return Request.CreateErrorResponse(HttpStatusCode.NotFound, "Record Not Found");
```
Comment (anebhargavkrishna) [inline DigiOrderApi/DigiOrderApi/Controllers/MenuItemController.cs:35]: Change status code to internal server error and message as Unable to process request

File: DigiOrderApi/DigiOrderApiDataAccess/MenuItemDataModel.cs [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+{
+    public class MenuItemDataModel
+    {
+        DataTable dtResult = null;
+
+        //Author: praveen kumar
+        //07-01-2020
+        //Description: Post a MenuItem
+        //Reviewer
+        //Review Version:1.0
```
Comment (anebhargavkrishna) [inline DigiOrderApi/DigiOrderApiDataAccess/MenuItemDataModel.cs:16]: Remove Commets

File: DigiOrderApi/DigiOrderApi/Controllers/MenuItemController.cs (hunk 1 above)
Comment (praveen957) [reply DigiOrderApi/DigiOrderApi/Controllers/MenuItemController.cs:20]: Reviewer is add

File: DigiOrderApi/DigiOrderApiDataAccess/MenuItemDataModel.cs (hunk 3 above)
Comment (praveen957) [reply DigiOrderApi/DigiOrderApiDataAccess/MenuItemDataModel.cs:16]: comments are removed

File: DigiOrderApi/DigiOrderApi/Controllers/MenuItemController.cs (hunk 2 above)
Comment (praveen957) [reply DigiOrderApi/DigiOrderApi/Controllers/MenuItemController.cs:35]: string and status code are changed as you told

**Pull request merged**